SchoolProject/src/main/java/Main.java

```diff
@@ -1,54 +1,123 @@
+import models.Display;
 import models.School;
 import repositories.StudentRepository;
+import repositories.TeacherRepository;
 import services.AccessValidator;
 import services.ReadDataService;
 import services.WriteDataService;
+import javax.swing.JOptionPane;
 
-import java.util.Scanner;
+import helpers.Grade;
+import helpers.Score;
 
 public class Main {
     public static void main(String args[]) {
```

Comment:
This main method is very big, so, probably you can continue refactoring, maybe create a ScannerManager that receives the input from the user